### ISSUE TITLE
App won't connect.

### ISSUE THREAD

**Issue by jrwrodgers** [OnewheelCommunityEdition/OWCE_App#74]
Connect pint to FM app. Force stop FM app. Connect with OWCE app.. message arrives to advise connection with FM app first, app then dies.

Pint firmware version Gemini 5059

**Comment by beeradmoore** [comment]
Hey @jrwrodgers. As far as I am aware there is nothing in the 5059 firmware preventing this.

There is an is a small window of time between when the FM app initially connects to the board through to when OWCE connects. I don't recall what this time is but I can do some testing to find out. 

The sooner you you can swap between the apps from when you see data normally load (usually when you see activated appear in the ride mode area is good indicator) the better the chances are of getting in that window.

Is it possible to jump between them quicker and see if this fixes it? I'll see what I can do internally to help users jump between these hoops faster.

**Comment by jrwrodgers** [comment]
hi @beeradmoore, Thanks for getting back to me on this. I would say that once I see the battery voltage and ride mode appear I close the FM app and try connecting with the OWCE app within 5 seconds. Not sure I can do this any faster. This is on an android phone Samsung S10.

**Comment by beeradmoore** [comment]
Yep, sounds like you are leaving FM OW app as early as possible. When you swap back how long is it taking OWCE to connect? I wonder if I can help by improving that connection speed (and/or connect automatically when we come back to OWCE) to help get inside that window.

**Comment by mstraa** [comment]
Hi ! I have the same issue, I tried a lot differents combinaisons but can't connect the app to my Pint X :(

HW: 7314
FW: Gemini 5076
